js/src/lib/interfaces/Types.ts

```diff
@@ -718,7 +718,7 @@ export interface ModelData {
 		inference?: boolean | {
 			parameters?: Record<string, any>;
 		};
-		[key: string]?: unknown;
+		[key: string]: unknown;
```

Comment:
I got some linting errors here, but tbh I don't really know what is going on here @pcuenca does this change look good to you?

Reply:
The optional looked better on paper, but I'm not sure either. How are you testing? Are you using github codespaces or something else?